connectivity/check/wait.go

```diff
@@ -53,9 +53,13 @@ func WaitForDeployment(ctx context.Context, log Logger, client *k8s.Client, name
 }
 
 // WaitForCiliumEndpoint waits until the specified cilium endpoint gets created.
-func WaitForCiliumEndpoint(ctx context.Context, log Logger, client *k8s.Client, namespace, name string) error {
-	log.Logf("⌛ [%s] Waiting for CiliumEndpoint for pod %s/%s to appear...", client.ClusterName(), namespace, name)
+func WaitForCiliumEndpoint(ctx context.Context, log Logger, client *k8s.Client, namespace string, ciliumEndpointDisabled bool, name string) error {
+	if ciliumEndpointDisabled {
```

Comment:
Adding a flag which makes this function a no-op doesn't make much sense. Checking this bool should be the responsibility of the caller, not the function.

Consider the following

```
func doSomething(dontDoIt bool) {
   if dontDoIt {
       return
   }
   actuallyDoIt()
}
```

compared to

```
func doSomething() {
   actuallyDoIt()
}

// ...

if !dontDoIt {
   doSomething()
}
```

Regardless of how we discover the value of `ciliumEndpointDisabled`. It should not be a parameter of this function.

Reply:
Fixed.